src/MICore/LaunchOptions.cs

```diff
@@ -1267,7 +1267,7 @@ public static LaunchOptions GetInstance(HostConfigurationStore configStore, stri
             {
                 try
                 {
-                    JObject parsedOptions = JObject.Parse(options);
+                    JObject parsedOptions = JsonConvert.DeserializeObject<JObject>(options, new JsonSerializerSettings { DateParseHandling = DateParseHandling.None });
```

Comment:
I think you need one tweak here:

```suggestion
                    JObject parsedOptions = JsonConvert.DeserializeObject<JObject>(options, new JsonSerializerSettings { DateParseHandling = DateParseHandling.None });
                    if (parsedOptions is null)
                    {
                        throw new InvalidLaunchOptionsException(MICoreResources.Error_UnknownLaunchOptions);
                    }
```